packages/phpstan-rules/src/Rules/Explicit/SameNamedParamFamilyRule.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\PHPStanRules\Rules\Explicit;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\ClassMethod;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\ClassReflection;
+use PHPStan\Reflection\Php\PhpMethodReflection;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Extension\ConfigurationExtensionInterface;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+use Symfony\Component\HttpKernel\Kernel;
+use Symplify\Astral\Naming\SimpleNameResolver;
+use Symplify\PHPStanRules\ParentGuard\ParentElementResolver\ParentMethodResolver;
+use Symplify\PHPStanRules\Rules\AbstractSymplifyRule;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Symplify\PHPStanRules\Tests\Rules\Explicit\SameNamedParamFamilyRule\SameNamedParamFamilyRuleTest
+ */
+final class SameNamedParamFamilyRule extends AbstractSymplifyRule
+{
+    /**
+     * @var string
+     */
+    public const ERROR_MESSAGE = 'Named arguments must share the same name with parent class method to allow using named arguments';
```

Comment:
IMO it would be great to name actual vs expected param name in the error message for good DX.
Otherwise the dev need to lookup these things by hand

Reply:
I was thinking about simple ignored messages here, but those can be handled by regex.

Thanks for the tip, I'll update it 👍

Reply:
https://github.com/symplify/symplify/pull/3430